packages/contracts-ops/contracts/scripts/Reboot.s.sol

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity 0.7.6;
+pragma abicoder v2;
+
+/*//////////////////////////////////////////////////////////////
+IMPORTS
+//////////////////////////////////////////////////////////////*/
+
+// Utilities
+import {Script, console2} from "forge-std/Script.sol";
+
+// Ops libs
+import {Config} from "../Config.sol";
+import {CallBatch} from "../CallBatch.sol";
+import {DeployAccountantLogic} from "./DeployAccountant.s.sol";
+import {DeployImplementationsLogic} from "./DeployImplementations.s.sol";
+import {InitializeImplementationsLogic} from "./InitializeImplementations.s.sol";
+import {UpgradeCallBatchLogic} from "./UpgradeCallBatches.s.sol";
+import {RotateUpdaterLogic} from "./RotateUpdater.s.sol";
+import {EnrollReplicasLogic} from "./EnrollReplica.s.sol";
+
+// Contract for selector
+import {UpgradeBeaconController} from "@nomad-xyz/contracts-core/contracts/upgrade/UpgradeBeaconController.sol";
+
+contract RebootLogic is
+    Config,
+    CallBatch,
+    RotateUpdaterLogic,
+    EnrollReplicasLogic,
+    DeployAccountantLogic,
+    DeployImplementationsLogic,
+    InitializeImplementationsLogic,
+    UpgradeCallBatchLogic
+{
+    function reboot(string memory _domain, string memory _configPath) internal {
+        // if ethereum, deploy accountant setup
+        if (keccak256(bytes(_domain)) == keccak256(bytes("ethereum"))) {
```

Comment:
needs a check that accountant is not already deployed?

Reply:
https://github.com/nomad-xyz/monorepo/pull/511/commits/fe939978e924efcda92c6abb73c4d7444a0324b0